src/engine/client/cl_main.cpp

```diff
@@ -727,6 +727,8 @@ void CL_ShutdownAll()
 		void SV_ShutdownGameProgs();
 		SV_ShutdownGameProgs();
 
+		// free old stuff
+		CM_FreeAll();
```

Comment:
There is no CM_LoadMap in the client so this probably shouldn't be here (that goes for the existing CM_ClearMap as well)

Reply:
I removed both.